be/src/olap/memtable_memory_limiter.cpp

```diff
@@ -27,6 +27,7 @@ namespace doris {
 DEFINE_GAUGE_METRIC_PROTOTYPE_5ARG(memtable_memory_limiter_mem_consumption, MetricUnit::BYTES, "",
                                    memtable_memory_limiter_mem_consumption,
                                    Labels({{"type", "load"}}));
+bvar::Adder<int64_t> g_flush_async_num("memtable_flush_async_num");
```

Comment:
what knowledge can we get from this bvar?

Reply:
Redundant, just for testing, I'll rm it.